{{ cookiecutter.dir_name }}/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
+    }
+}
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
```

Comment:
Hm, I suppose we could look inside `$ANDROID_HOME/build-tools/` if ANDROID_HOME is already set, but I think it's important to have a fallback and not to enforce that (since it may be run through briefcase, and before user has ANDROID_HOME set).

Also, Gradle downloads the version indicated if it doesn't find it in the system, so it might not be such a bad idea for reproducibility to leave it hardcoded at first. :)